### PR TITLE
Add missing changelog for #5331

### PR DESCRIPTION
**Pull request by jdannberg**

### CODE REVIEW

File: doc/modules/changes/20240614_dannberg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Changed: The operator splitting scheme can now use SUNDIALs
+to compute the reactions (rather than a hand-written
+forward Euler scheme). ARKode is the new default, but the
```
Comment (gassmoeller) [inline doc/modules/changes/20240614_dannberg:3]: Can you rephrase this a bit to make clear that ARKode is a part of SUNDIALs?

Comment (jdannberg) [reply doc/modules/changes/20240614_dannberg]: Better?

**Review by gassmoeller — APPROVED** [review]
